### ISSUE TITLE
provide the IP of the nextbox host to the user

### ISSUE THREAD

**Issue by jerabaul29** [Nitrokey/nextbox-daemon#16]
I have activated the ssh access control (Nextbox App > System settings > SSH access control).

The IP address on which the RPi of my nextbox is available is not provided in the web interface. That is not a big deal, I just needed to do a bit of ```nmap``` to find it, but it is one extra step that I would happily skip :) .

Would it be possible to display the IP on which to ssh to access the nextbox, and maybe even better, provide the full

```ssh nextuser@NEXTBOX_IP```

command in the web gui, so that it can be simply copied and pasted in a terminal to ssh on it? :)

**Comment by daringer** [comment]
Yup, good one,
but providing only the local IP and user to use for the connection.
Showing the command-line is not OS independent, means no added value/confusing for some users.

**Comment by jerabaul29** [comment]
Yeah, good point about the command being OS dependent. I am just very Linux centric ^^ :) .
